Gemfile.lock

```diff
@@ -231,5 +231,8 @@ DEPENDENCIES
   web-console (>= 3.3.0)
   webpacker
 
+RUBY VERSION
+   ruby 2.4.0p0
```

Comment:
This was introduced when running `bundle install`. It should have already been there but for some reason wasn't. For more info https://devcenter.heroku.com/articles/ruby-versions#ruby-version-specifiers